app/src/main/java/ch/epfl/sweng/SDP/auth/Account.java

```diff
@@ -343,13 +342,13 @@ public void increaseTotalMatches() throws DatabaseException {
 
     /**
      * Method that allows one to change the average rating per game given a new rating.
-     * The rating passed as parameter should be the average rating obtained after a match.
+     * The rating passed as parameter should be the total rating obtained after a match.
      *
-     * @throws IllegalArgumentException in case a rating <= 0 or > 5 is given
+     * @throws IllegalArgumentException in case a rating <= 0 or > 25 is given
      * @throws DatabaseException        in case write to database fails
      */
     public void changeAverageRating(double rating) throws DatabaseException {
-        checkPrecondition(0 < rating && rating <= 5, "Wrong rating given");
+        checkPrecondition(0 < rating && rating <= 25, "Wrong rating given");
 
```

Comment:
Actually this should be 20 since you cannot vote for yourself

Reply:
You’re right, thanks for spotting it out!